Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3061,6 +3059,16 @@ static FloatRect computeMarqueeSelectionRect(const WebCore::FloatPoint& point1,
         return { scrollDeltaWidth, scrollDeltaHeight };
     }();
 
+#if PLATFORM(MAC)
+    IntSize scrollAdjustmentBasedOnScreenBoundaries = { 0, 0 };
```

Comment:
Nit: it's okay to not explicitly initialize here. The default member initializers will do the right thing for us.
```suggestion
    IntSize scrollAdjustmentBasedOnScreenBoundaries;
```